internal/pkg/buildcfg/config_darwin.go

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package buildcfg
+
+var (
+	// PACKAGE_NAME contains the name of the package
+	PACKAGE_NAME string = "singularity"
+	// PACKAGE_VERSION contains the version of the package
+	PACKAGE_VERSION string = "master"
```

Comment:
This is working around the issue that `mconfig` doesn't produce a working `builddir` on macOS.  I don't recommend merging this as-is - just trying to get back to a working state and can also work more on the `mconfig` changes needed for macOS support.